avocado/core/test.py

```diff
@@ -383,6 +384,11 @@ def _start_logging(self):
                                                        formatter,
                                                        self._ssh_logfile)
 
+        if isinstance(sys.stdout, output.LoggingFile):
+            sys.stdout.add_logger(logging.getLogger("avocado.test.stdout"))
+        if isinstance(sys.stderr, output.LoggingFile):
+            sys.stderr.add_logger(logging.getLogger("avocado.test.stderr"))
```

Comment:
So the proper fix is actually to add `avocado.test.stdout` handler to `output.LoggingFile` when the `file_handler` is added there and the same for stderr.

On the other hand, we could simply add `NullHandler` to `avocado.test.stdout` and `avocado.test.stderr` when it's defined in `avocado.core.output`:

```diff
diff --git a/avocado/core/output.py b/avocado/core/output.py
index fdeab5e..3a0f935 100644
--- a/avocado/core/output.py
+++ b/avocado/core/output.py
@@ -403,6 +403,8 @@ def reconfigure(args):
     app_logger.addHandler(app_err_handler)
     app_logger.propagate = False
     if not os.environ.get("AVOCADO_LOG_EARLY"):
+        logging.getLogger("avocado.test.stdout").addHandler(NULL_HANDLER())
+        logging.getLogger("avocado.test.stderr").addHandler(NULL_HANDLER())
         logging.getLogger("avocado.test.stdout").propagate = False
         logging.getLogger("avocado.test.stderr").propagate = False
         if "early" in enabled:
```